docs/SVG.md

```diff
@@ -0,0 +1,87 @@
+# Scalable Vector Graphics (SVG) #
+
+`fpdf2` supports basic conversion of SVG paths into PDF paths, which can be
+inserted into an existing PDF document or used as the contents of a new PDF
+document.
+
+Not all SVGs will convert correctly. Please see
+[the list of unsupported features](#currently-unsupported-notable-svg-features)
+for more information about what to look out for.
+
+## A simple example ##
+
+The following script will create a PDF that consists only of the graphics
+contents of the provided SVG file:
+
+```python
+import fpdf
+
+svg = fpdf.svg.SVGObject.from_file("my_file.svg")
+
+pdf = fpdf.FPDF(unit="pt", format=(svg.width, svg.height))
+pdf.add_page()
+svg.draw_to_page(pdf)
+
+pdf.output("my_file.pdf")
+```
+
+This does assume that the width/height of the SVG are specified in absolute
+units rather than relative ones (i.e. the top-level `<svg>` tag has something
+like `width="5cm"` and not `width=50%`).
```

Comment:
I think I need to update this documentation, since I ended up adding some basic best-effort percentage handling, where a `width=50%` SVG should end up being scaled to to half of the page width.

Reply:
I updated the documentation to elaborate a bit further on the behavior in this case.